src/Land/Model/SoilHeatParameterizations.jl

```diff
@@ -0,0 +1,192 @@
+"""
+    SoilHeatParameterizations
+
+Functions for volumetric heat capacity, temperature as a function
+of volumetric internal energy, saturated thermal conductivity, thermal
+conductivity, relative saturation and the Kersten number are included.
+All heat capacities are volumetric.
+"""
+module SoilHeatParameterizations
+
+using DocStringExtensions
+
+export volumetric_heat_capacity,
+    volumetric_internal_energy,
+    saturated_thermal_conductivity,
+    thermal_conductivity,
+    relative_saturation,
+    kersten_number,
+    volumetric_internal_energy_liq,
+    temperature_from_U
+
+
+"""
+    function temperature_from_U(
+        T_ref::FT,
+        U::FT,
+        θ_i::FT,
+        ρ_i::FT,
+        LH_f_0::FT,
+        cs::FT
```

Comment:
While on the lines of confusing names/consistency, the heat capacity variables here probably should get changed....

from the naming page:https://clima.github.io/ClimateMachine.jl/latest/DevDocs/VariableList/

cp_X and cv_X denote heat capacities for constant P or constant V processes. we're using cp_X for volumetric heat capacity (and "cs" here, for the volumetric heat capacity of the soil, and "c_ds" for the bulk volumetric heat capacity of the dry soil).  What about: `c_vol_x` instead? (note that the parameters we get from param_set are cp_X, but when we multiply them by rho, we should rename those variables to be clearer). If we do `_vol_` here, should we be consistent and use `E_vol_int` for volumetric internal energy?

Reply:
ah ok! I think it might kind of the opposite in this case, but using rho in the name makes sense -  `cp_l` is a specific heat capacity (J/kg/K), and we want J/vol/K, so calling it `\rhocp_l` would make sense.

Reply:
made this fix